appendix/basic_commands.tex

```diff
@@ -1498,24 +1498,24 @@ \subsection{GET}
 \subsection{GET\#}
 
 The {\ttfamily GET\#} statement reads data from a specified logical file one
-character at a time.  It is identical to the {\ttfamily GET} statement except
-that it requires a first argument specifies a logical file identifier.  The
+character at a time.  It is comparable ***SIMILAR?*** to the {\ttfamily GET} statement although 
+it requires a first argument that specifies a logical file identifier.  The
```

Comment:
I prefer the existing wording, but with a quick grammatical fix.  The two statements are identical, with the exception of the device argument.  The word "although" conveys this as if it's an alternative or side note rather than a strict requirement...which is what it is.

I see I misuse "specifies" here, but I think this could be rephrased "It is identical to the GET statement except that in requires a first argument to specify a logical file identifier."

Reply:
Try 

It is similar to the GET statement, except that the first argument is the logical file number (LFN).

Reply:
I was just thinking "identical" with slight differences wasn't identical.  I'm okay with what you consider correct.